docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -85,7 +85,9 @@ ifdef::context[:parent-context: {context}]
 
 // Purpose statement for the assembly
 [role="_abstract"]
-As an owner of a Kafka instance in {product-long-kafka}, you can manage the level of access that other user accounts and service accounts have to your instance. You can allow or deny access to your instance for specific accounts or for all accounts in your organization. You can also allow other users or service accounts to manage the level of access to your instance for you.
+As an owner of a Kafka instance in {product-long-kafka}, you control immutable access permissions on all resources within that Kafka instance including topics, groups, transactional IDs, and Access Control Lists (ACLs). These permissions are implicit meaning that other users do not see them listed in the {product-long-kafka} web console or from Kafka APIs.
```

Comment:
A couple of comments:

- _Immutable_ means "cannot be changed". I _think_ Keith meant that the level of control an owner has by default is immutable, not that access permissions are immutable. Access permissions _can_ be changed, for example, to grant access to other accounts.
- I don't think we need to introduce a term such as _implicit,_ which then requires definition. We can probably say this in plainer language. 

Therefore, I suggest a rewrite like the following:

```
As an owner of a Kafka instance in {product-long-kafka}, you control access permissions for all resources within that Kafka instance by default. This includes topics, groups, transactional IDs, and Access Control Lists (ACLs) themselves. Other users cannot see permissions for your Kafka instance in the {product-long-kafka} web console or from Kafka APIs.
```